3rdParty/rocksdb/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ set(ARANGO_ROCKSDB_VERSION "v5.6.X")
 # jemalloc settings
 if (USE_JEMALLOC)
```

Comment:
why is it within USE_JEMALLOC? please put it outside. it might be coupled now but that might change